Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        //Calls all user prompt methods & prints full message to console
+        public static void Demo()
+        {
+            string name = PromptName();
```

Comment:
Demo could call ProcessPerson method instead of all those lines. Also, second person is missing.